accessibility_monitoring_platform/apps/notifications/templates/notifications/task_list.html

```diff
@@ -24,90 +18,121 @@ <h1 class="govuk-heading-xl amp-margin-bottom-30">{{ sitemap.current_platform_pa
+                    <li class="amp-custom-nav-bar amp-margin-right-25 {% if type == 'reminder' %} amp-custom-nav-bar-selected {% endif %}">
+                        <div class="amp-margin-bottom-15">
+                            <a href="?type=reminder" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">Reminder ({{ task_type_counts.reminder }})</a>
+                        </div>
                     </li>
-                    <li>
-                        {% if type == 'overdue' %}
-                            <b>Overdue ({{ task_type_counts.overdue }})</b>
-                        {% else %}
-                            <a href="?type=overdue" class="govuk-link govuk-link--no-visited-state">
-                                Overdue ({{ task_type_counts.overdue }})</a>
-                        {% endif %}
+                    <li class="amp-custom-nav-bar amp-margin-right-25 {% if type == 'overdue' %} amp-custom-nav-bar-selected {% endif %}">
+                        <div class="amp-margin-bottom-15">
+                            <a href="?type=overdue" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">Overdue ({{ task_type_counts.overdue }})</a>
+                         </div>
                     </li>
-                    <li>
-                        {% if type == 'postcase' %}
-                            <b>Post case ({{ task_type_counts.postcase }})</b>
-                        {% else %}
-                            <a href="?type=postcase" class="govuk-link govuk-link--no-visited-state">
-                                Post case ({{ task_type_counts.postcase }})</a>
-                        {% endif %}
+                    <li class="amp-custom-nav-bar {% if type == 'postcase' %} amp-custom-nav-bar-selected {% endif %}">
+                        <div class="amp-margin-bottom-15">
+                            <a href="?type=postcase" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">Post case ({{ task_type_counts.postcase }})</a>
+                         </div>
                     </li>
                 </ul>
             </div>
-            <div class="govuk-grid-column-one-half">
-                <ul class="govuk-list">
-                    <li>
-                        {% if show_all_users %}
-                            <b>Showing tasks for all users</b>
-                        {% else %}
-                            <a href="?show_all_users=true" class="govuk-link govuk-link--no-visited-state">
-                                Show tasks for all users</a>
-                        {% endif %}
-                    </li>
-                </ul>
+            <div class="govuk-grid-column-full">
+                {% if type == None %}
+                    <ul class="govuk-list">
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if type == None and show_all_users == None %}
```

Comment:
`type` has to be `None` to reach this code, so the condition could be `if show_all_users == None`.

This is more commonly written as `if not show_all_users` (or even `if show_all_users` render link `else` `<b>All</b>).

I don't think that this is behaving quite as you intend because the `type` parameter is lost from context when `show_all_users` is set:

<img width="773" height="519" alt="image" src="https://github.com/user-attachments/assets/6937f39a-c8ce-4580-8316-e28af39a5399" />

This appears to be a bug in the view.

Reply:
Replaced `== None` with `if not`